src/patternfly/utilities/variables.scss

```diff
@@ -67,12 +67,25 @@ $pf-global--dark-color--dark:    $pf-color-black-800 !default;
 
 
 // Shadows
-$pf-global--sm--BoxShadow:     0  pf-prem(2)  pf-prem(3) pf-prem(-2) rgba($pf-color-black-1000, .25) !default;
-$pf-global--md--BoxShadow:     0  pf-prem(2)  pf-prem(2) rgba($pf-color-black-1000, .2) !default;
-$pf-global--lg--BoxShadow:     0  pf-prem(3)  pf-prem(5) rgba($pf-color-black-1000, .3) !default;
-$pf-global--inset--BoxShadow:  inset 0  pf-prem(1)  pf-prem(2) rgba($pf-color-black-1000, .1) !default;
-$pf-global--left--BoxShadow:   -.2rem 0 .5rem -.2rem rgba($pf-color-black-1000, .3) !default;
 
+$pf-global--sm--BoxShadow:  0  pf-prem(1)  pf-prem(2) 0 rgba($pf-color-black-1000, .2) !default;
```

Comment:
We don't have a `pf-prem` function anymore. now it's 2 functions:

- pf-size-prem
- pf-font-prem

Depending if you want to set up a size for font or space. In this case I'd use the one for size.